meta-facebook/yv35-rf/src/platform/plat_sdr_table.c

```diff
@@ -81,7 +81,7 @@ SDR_Full_sensor plat_sdr_table[] = {
 		0x00, // reserved
 		0x00, // OEM
 		IPMI_SDR_STRING_TYPE_ASCII_8, // ID len, should be same as "size of struct"
-		"RF_CXL_TEMP_C",
+		"RF CXL_Cntr Temp",
```

Comment:
This sensor name should be consistent with our common sensor naming spec.  The name should be something like "RF_CXL_CNTR_TEMP_C"

Reply:
I have modified this sensor name in the #62b93e commit.
Many thanks.